### ISSUE TITLE
MTProxybot has a bug and does not show me my proxies

### ISSUE THREAD

**Issue by EmadHelmi** [TelegramMessenger/MTProxy#22]
After I run the commands step by step from your git, when i create a proxy in the `@MTProxybot` it said everything is ok but when I want to get the list of my proxies I see [this](https://ibb.co/jgbYnd) result from the bot. When i click in the blank it says invalid proxy. I think it is a bug

**Comment by seega** [comment]
try latest telegram client

**Comment by Badiboy** [comment]
+1, the same problem. Appears once for one added proxy. It worked before and works after, but I had ine "empty" button. Button is empty both on desctop and android.

**Comment by riot26** [comment]
Same problem. [Screenshot](http://i.imgur.com/78v1xjw.png)

**Comment by ulyanchesnokov** [comment]
Same problem. I have one empty field and one worked well with the same domain and another port.

![default](https://user-images.githubusercontent.com/2368195/40803859-5b9d36aa-6522-11e8-865e-495603d756d3.png)

**Comment by belko-letyago** [comment]
Same problem. Both desktop client(1.2.20) and Android (4.8.9) show empty button.
![image](https://user-images.githubusercontent.com/3055972/40831470-c06725e0-6591-11e8-9962-eb3bbf0695c5.png)

**Comment by seega** [comment]
Now i have this problem: /stats freeze and promo not displayed, docker restart did not help, i try delete and add proxy with same ip/port/secret, now button is empty, but new tag generated and promo work from the previous configuration.

**Comment by Badiboy** [comment]
I see that the issue is fixed now.

Stat still show nothing...

**Comment by EmadHelmi** [comment]
@Badiboy fixed?!! I stiil have the same problem

**Comment by belko-letyago** [comment]
@EmadHelmi My bug with the empty button was fixed too:
![image](https://user-images.githubusercontent.com/3055972/40865000-e2227604-65fe-11e8-8b6b-d34bd79f535a.png)

**Comment by seega** [comment]
@EmadHelmi my empty button removed (tg desktop 1.30)

**Comment by Badiboy** [comment]
![image](https://user-images.githubusercontent.com/5613421/40865061-30f26c26-65ff-11e8-9f77-d432e76c1902.png)
May be the reason was that I had updated the Desctop client to version 1.3?...

**Comment by belko-letyago** [comment]
@Badiboy Nope. I am still on 1.2.24 and issue is fixed. It was on the server side, not a client.
P.S. Thanks for the news about 1.3 version release :), I have missed it.

**Comment by EmadHelmi** [comment]
@belko-letyago @Badiboy If i create a new mtp with a new port on the same machine and add it to the bot it shows me that mtp but if i create again the same port mtp on the bot it does not show me, i think i must create my mtp on a different port :(
![0a8f65dd-7993-4628-94da-9dec68784a9c](https://user-images.githubusercontent.com/21260959/40870287-b4371432-6641-11e8-997b-147a40c463a0.jpeg)

**Comment by mothlike** [comment]
If i create a proxy and delete it and then create it again, it successfully created with the same tag, but not appears in the list of /myproxies. I reproduced it with several servers and now want them back in the proxies list.
